backend/manager/modules/dal/src/main/resources/bundles/AuditLogMessages.properties

```diff
@@ -1337,7 +1337,7 @@ VM_BACKUP_SUCCEEDED=Backup ${backupId} for VM ${VmName} completed successfully (
 VM_BACKUP_FAILED=Backup ${backupId} for VM ${VmName} failed (User: ${UserName}).
 VM_BACKUP_FINALIZED=Backup ${backupId} for VM ${VmName} finalized (User: ${UserName}).
 VM_BACKUP_FAILED_TO_FINALIZE=Backup ${backupId} for VM ${VmName} failed to finalize (User: ${UserName}).
-VM_BACKUP_STOPPED=Backup ${backupId} for VM ${VmName} stopped (User: ${UserName}).
+VM_BACKUP_STOPPED_BEFORE_READY=Backup ${backupId} for VM ${VmName} stopped before reaching READY state (User: ${UserName}).
```

Comment:
does it make a difference for users that the backup is stopped before reaching the 'ready' state? if not, let's not adding a different message for this case to reduce the translation effort